state/upgrades.go

```diff
@@ -4581,3 +4581,40 @@ func CharmOriginChannelMustHaveTrack(pool *StatePool) error {
 		return nil
 	}))
 }
+
+// RemoveDefaultSeriesFromModelConfig removes the default series value from
+// each model's config. To allow users to set the value and be sure it's what
+// they want. Previously it was set by juju.
+func RemoveDefaultSeriesFromModelConfig(pool *StatePool) error {
+	st, err := pool.SystemState()
+	if err != nil {
+		return errors.Trace(err)
+	}
+	var ops []txn.Op
+	coll, closer := st.db().GetRawCollection(settingsC)
+	defer closer()
+	pattern := fmt.Sprintf(":%s$", modelGlobalKey)
```

Comment:
I never fully understood why we have this "e" (`modelGlobalKey`).

Reply:
@cderici My best guess would be a hold over from juju 1.x, where models didn't exist and we had 'environ'?